lib/Elements/Element.tsx

```
@@ -0,0 +1,373 @@
+import React, { memo, useMemo } from "react";
+import { View, Text, ScrollView } from "react-native";
+import { useStreamDown } from "../context";
+import type { StreamdownToken } from "../marked";
+import Slot from "./Slot";
+import Link from "./Link";
+import Image from "./Image";
+import Math from "./Math";
```

Biome (2.1.2)

[error] Do not shadow the global "Math" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)